docs/main/kafka-knative-broker/README.md

```diff
@@ -0,0 +1,103 @@
+# Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfills this reliability need.
+
+## Pre-requisites
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+
+## Installation steps
+
+1. Configure and enable kafka broker feature in Knative: https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/
+  * i.e: 
+```console
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-controller.yaml
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-broker.yaml
+```
+> [!NOTE]
+> We are using the version `knative-v1.14.5` in this example, this may change, please refer to [the official documentation link](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/)
```

Comment:
nit: use the active form to improve clarity.

e.g. `This example uses the version `knative-v1.14.5`, this may change, please refer to ...`